lib/transaction/transaction.js

```diff
@@ -878,8 +883,7 @@ Transaction.prototype.getFee = function () {
  */
 Transaction.prototype._estimateFee = function () {
   var estimatedSize = this._estimateSize()
-  var available = this._getUnspentValue()
-  return Transaction._estimateFee(estimatedSize, available, this._feePerKb)
+  return Math.ceil(estimatedSize / 1000 * (this._feePerKb || Transaction.FEE_PER_KB))
```

Comment:
why remove the estimator for the change output? is that related to your above changes?

Reply:
We no longer need a change estimator, because we are using a dummy change output from above.